mitodl/open-discussions
# Added opengraph metadata tags for social sharing

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1872

#### What's this PR do?
Adds some default metadata tags that are used by Facebook to show a coherent link preview if an Open Discussions page is shared

#### How should this be manually tested?
https://developers.facebook.com/tools/debug/sharing/

Enter any URL for the PR build (https://odl-open-discussions-c-pr-1872.herokuapp.com/). The
tool should show a preview that matches the screenshot below

#### Any background context you want to provide?
This PR just adds some default metadata tags that will show for all pages in the app. The linked issue describes specific metadata for specific pages, but those requirements will be moved to a new issue

#### Screenshots (if appropriate)
![ss 2019-02-22 at 13 41 21](https://user-images.githubusercontent.com/14932219/53263690-2b472800-36a8-11e9-983b-763103568bdd.png)
